backend/api/Controllers/MissionController.cs

```diff
@@ -349,6 +350,8 @@ public async Task<ActionResult<byte[]>> GetMap([FromRoute] string installationCo
             [FromBody] ScheduledMissionQuery scheduledMissionQuery
         )
         {
+            // TODO: once we have a good way of creating mission definitions for echo missions,
+            //       we can delete this endpoint
```

Comment:
Instead of creating a TODO comment I would suggest removing the comment and opening an issue instead where you link to this specific line.

Reply:
I agree. This comment was deleted at some point since it is not relevant anymore, but it seems to have been re-introduced in a rebase somewhere. Another PR already addresses it